packages/radix-ui-themes/src/components/data-list.tsx

```diff
@@ -0,0 +1,81 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import { Text } from './text';
+import { extractProps, marginPropDefs } from '../helpers';
+import { dataListPropDefs, dataListItemPropDefs, dataListLabelPropDefs } from './data-list.props';
+import type { MarginProps, GetPropDefTypes } from '../helpers';
+
+type DataListRootElement = HTMLDListElement;
+type DataListRootOwnProps = GetPropDefTypes<typeof dataListPropDefs>;
+interface DataListRootProps
+  extends React.ComponentPropsWithoutRef<'dl'>,
+    MarginProps,
+    DataListRootOwnProps {}
+const DataListRoot = React.forwardRef<DataListRootElement, DataListRootProps>(
+  (props, forwardedRef) => {
+    const { className, ...dataListProps } = extractProps(props, dataListPropDefs, marginPropDefs);
+    return (
+      <Text asChild>
+        <dl
+          {...dataListProps}
+          ref={forwardedRef}
+          className={classNames(className, 'rt-DataListRoot')}
+        />
+      </Text>
+    );
+  }
+);
+DataListRoot.displayName = 'DataListRootGrid';
```

Comment:
Any reason we'd use a different display name?

Reply:
Good catch, this was an oversight!